arch/arm64/kvm/hyp/nvhe/debug/hyp-dbg.c

```diff
@@ -50,6 +51,9 @@ int hyp_dbg(u64 cmd, u64 param1, u64 param2, u64 param3, u64 param4)
 	case 3:
 		count_shared(param1, param2, param3);
 		break;
+    case 4:
```

Comment:
The indentation is broken. The indentation is supposed to be a tab, which equals 8 spaces, here it is 4 spaces.